src/Components/SmartComponents/Dashbar/Dashbar.js

```diff
@@ -8,6 +8,43 @@ import { useDispatch } from 'react-redux';
 import { changeCveListParameters } from '../../../Store/Actions/Actions';
 import { FormattedMessage } from 'react-intl';
 import messages from '../../../Messages';
+import propTypes from 'prop-types';
+
+const DashbarItem = ({ title, number, impact, onClick }) => {
+    return (
+        <Card isFullHeight className="card-box">
+            <CardBody>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold">
+                    {title && title}
+                </Text>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold pf-u-mt-xs">
+                    <a onClick={onClick}>
+                        {impact === 'critical'
```

Comment:
This is a bit of hardcoding, what you could do is go to `constants.js` and find `impactList` constant. Add new property to critical and important called `iconColor` where you can define this values and not hardcore this security icon two times.

Reply:
That sounds a lot better! I can propably use the color property, so the color is consistent through the whole app.

Reply:
color is already used for text, and text and icon color is not always the same for severity